colin-api/src/colin_api/models/reset.py

```diff
@@ -239,3 +239,49 @@ def reset_filings(cls, start_date: str = None, end_date: str = None, identifiers
             if con:
                 con.rollback()
             raise err
+
+    @classmethod
+    def reset_filings_by_event(cls, event_ids: list = []):
+        """Reset changes made for given event ids."""
+        # initialize reset object
+        reset_obj = Reset()
+
+        # place into lists that can be reset together
+        annual_report_events = []
+        events_info = []
+
+        for filing_info in reset_obj.get_filings_for_reset():
+            if filing_info['event_id'] in event_ids:
+                events_info.append(filing_info)
+                if filing_info['filing_typ_cd'] in Filing.FILING_TYPES['annualReport']['type_code_list']:
+                    annual_report_events.append(filing_info['event_id'])
+
+        try:
+            if event_ids:
+                # setup db connection
+                con = DB.connection
+                con.begin()
+                cursor = con.cursor()
+
+                # reset data in oracle for events
+                # The commented out events do not seem to happen for AR so they are commented out.
+                new_corps = cls._get_incorporations_by_event(cursor, event_ids)
+                #Party.reset_dirs_by_events(cursor=cursor, event_ids=event_ids)
```

Comment:
So there is no need to reset party info?

From an AR for a BC company I pulled up in COLIN, I can see that there are `CORP_PARTY` entries

![image](https://github.com/bcgov/lear/assets/6685223/b762c4e2-eea1-4d9f-b0b1-61231524aad4)

Reply:
I don't think it's needed from the filing I did no event id was logged in BAR that corresponded to a CORP_PARTY entry in COLIN. That said because of that ther eis no harm in adding it so I'll add it in